android/build.gradle

```diff
@@ -156,6 +156,7 @@ android {
       "**/libc++_shared.so",
       "**/libfbjni.so",
       "**/libreactnativejni.so",
+      "**/libcrypto.so",
```

Comment:
by this line, the app crashes on the release build, I think we should remove this line and use this solution https://github.com/numandev1/react-native-keys#2-files-found-with-path-libcryptoso 
which works on both debug and release build

Reply:
Greate, i will send 1 PR to revert this change, thanks